src/hipercow/util.py

```diff
@@ -14,14 +14,13 @@
 def find_file_descend(filename: str, path: str | Path) -> Path | None:
     path = Path(path)
     root = Path(path.anchor)
-
-    while path != root:
+    while True:
         attempt = path / filename
         if attempt.exists():
             return attempt.parent
         path = path.parent
-
-    return None
+        if path == root:
+            return None
```

Comment:
This now ensures we get at least one iteration, instead of incorrectly bailing out at the start, if we're already at the root of the path.